frontend/src/scenes/project-homepage/ProjectHomepage.tsx

```diff
@@ -1,71 +1,43 @@
 import { projectHomepageLogic } from 'scenes/project-homepage/projectHomepageLogic'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { FEATURE_FLAGS } from 'lib/constants'
-import { CompactList } from 'lib/components/CompactList/CompactList'
-import { asDisplay } from 'scenes/persons/PersonHeader'
-import { ProfilePicture } from 'lib/components/ProfilePicture'
-import { humanFriendlyDuration } from 'lib/utils'
 import { LemonButton } from 'lib/components/LemonButton'
-import { dayjs } from 'lib/dayjs'
-import { SessionPlayerDrawer } from 'scenes/session-recordings/SessionPlayerDrawer'
-import { sessionRecordingsTableLogic } from 'scenes/session-recordings/sessionRecordingsTableLogic'
-import { RecordingWatchedSource } from 'lib/utils/eventUsageLogic'
-
-interface RecordingRowProps {
-    recording: SessionRecordingType
-}
-
-function RecordingRow({ recording }: RecordingRowProps): JSX.Element {
-    const sessionRecordingsTableLogicInstance = sessionRecordingsTableLogic({ key: 'projectHomepage' })
-    const { openSessionPlayer } = useActions(sessionRecordingsTableLogicInstance)
-    return (
-        <LemonButton
-            fullWidth
-            className="recording-row"
-            onClick={() => {
-                openSessionPlayer(recording.id, RecordingWatchedSource.ProjectHomepage)
-            }}
-        >
-            <ProfilePicture name={asDisplay(recording.person)} />
-            <div className="recording-person-text-container" style={{ flexDirection: 'column', display: 'flex' }}>
-                <p className="recording-person-text">{asDisplay(recording.person)}</p>
-                <p className="recording-start-time">{dayjs(recording.start_time).fromNow()}</p>
-            </div>
-            <span className="recording-duration">{humanFriendlyDuration(recording.recording_duration)}</span>
-            <IconPlayCircle style={{ fontSize: '1.25rem' }} />
-        </LemonButton>
-    )
-}
+import { RecentRecordings } from './RecentRecordings'
+import { RecentInsights } from './RecentInsights'
+import { useWindowSize } from 'lib/hooks/useWindowSize'
 
 export function ProjectHomepage(): JSX.Element {
     const { dashboardLogic } = useValues(projectHomepageLogic)
-    const sessionRecordingsTableLogicInstance = sessionRecordingsTableLogic({ key: 'projectHomepage' })
-    const { sessionRecordingId, sessionRecordings, sessionRecordingsResponseLoading } = useValues(
-        sessionRecordingsTableLogicInstance
-    )
-    const { closeSessionPlayer } = useActions(sessionRecordingsTableLogicInstance)
     const { featureFlags } = useValues(featureFlagLogic)
     const { currentTeam } = useValues(teamLogic)
     const { dashboard } = useValues(dashboardLogic)
     const { showInviteModal } = useActions(inviteLogic)
     const { showPrimaryDashboardModal } = useActions(primaryDashboardModalLogic)
 
+    const [topListContainerWidth, setTopListContainerWidth] = useState<number>()
```

Comment:
Could we use useSize here to only keep track of the width of the container?

Reply:
Ooooo nice! didn't know that was a thing!